evil-collection-git-timemachine.el

```diff
@@ -0,0 +1,22 @@
+;;; evil-collection-git-timemachine.el --- Bindings for `git-timemachine' -*- lexical-binding: t -*-
```

Comment:
You can use evil-collection-free-keys (or some other file) as an example/template.

You'll want the license/copyright/maintainer/etc as the header.

Of course, leave yourself as the author. :)